package.json

```diff
@@ -73,7 +65,7 @@
 		"build": "npm run build:esm && npm run build:node",
 		"changeset": "npx @changesets/cli",
 		"ci:build": "npm run build",
-		"ci:lint": "npm run lint && npm run madge:circular",
+		"ci:lint": "npm --prefix ./packages/common run ci:lint",
```

Comment:
Running `npm run build` and `npm run test` from the monorepo root still works as expected, but running `npm run lint` results in an error:

<img width="859" height="131" alt="2025-11-14_14-45-22" src="https://github.com/user-attachments/assets/01a43f28-60ae-44ca-b01c-523ca80991c2" />

Would be good/more consistent if we could still trigger the lint from the monorepo root.